Failing Test Jetty 11 - ServletContextHandlerTest.testAsyncServletRequestListenerListener

Jetty 11 build keep failing with this
```
java.lang.AssertionError: 

Expected: <2>
     but: was <9>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
	at org.eclipse.jetty.servlet@11.0.9-SNAPSHOT/org.eclipse.jetty.servlet.ServletContextHandlerTest.testAsyncServletRequestListenerListener(ServletContextHandlerTest.java:1052)
```


https://jenkins.webtide.net/job/jetty.project/job/jetty-11.0.x/1026/testReport/

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -48,6 +48,8 @@ pipeline {
             container('jetty-build') {
               timeout(time: 120, unit: 'MINUTES') {
                 dir("${env.WORKSPACE}/buildy") {
+                  //cleanup all projects
+                  mavenBuild("jdk17", "clean -T4", "maven3")
                   mavenBuild("jdk17", "clean install -f jetty-integrations", "maven3")
                 }
               }
@@ -59,6 +61,8 @@ pipeline {
             container('jetty-build') {
               timeout(time: 120, unit: 'MINUTES') {
                 dir("${env.WORKSPACE}/buildy") {
+                  //cleanup all projects
+                  mavenBuild("jdk17", "clean -T4", "maven3")
                   mavenBuild("jdk17", "clean install -f jetty-ee9", "maven3")
                 }
               }
@@ -70,6 +74,8 @@ pipeline {
             container('jetty-build') {
               timeout(time: 120, unit: 'MINUTES') {
                 dir("${env.WORKSPACE}/buildy") {
+                  //cleanup all projects
+                  mavenBuild("jdk17", "clean -T4", "maven3")
                   mavenBuild("jdk17", "clean install -f jetty-ee10", "maven3")
                 }
               }
@@ -81,6 +87,8 @@ pipeline {
             container('jetty-build') {
               timeout(time: 120, unit: 'MINUTES') {
                 dir("${env.WORKSPACE}/buildy") {
+                  //cleanup all projects
+                  mavenBuild("jdk17", "clean -T4", "maven3")
                   mavenBuild("jdk17", "clean install -f jetty-home", "maven3")
                 }
               }
@@ -93,6 +101,8 @@ pipeline {
             container('jetty-build') {
               timeout(time: 120, unit: 'MINUTES') {
                 dir("${env.WORKSPACE}/buildy") {
+                  //cleanup all projects
+                  mavenBuild("jdk17", "clean -T4", "maven3")
                   mavenBuild("jdk17", "clean install -f tests", "maven3")
                 }
               }

